server/rpc.go

```diff
@@ -340,12 +342,20 @@ func coreSegMetadata(segData *net.SegData) (*core.SegTranscodingMetadata, error)
 		dur = 2 * time.Second // assume 2sec default duration
 	}
 
+	caps := core.CapabilitiesFromNetCapabilities(segData.Capabilities)
+	if caps == nil {
+		// For older broadcasters. Note if there are any orchestrator
+		// mandatory capabilities, segment verification will fail.
```

Comment:
Should "segment verification will fail" say something like "seg creds verification will fail" since I believe it is referring to the compatibility check in `verifySegCreds()`?

Reply:
Fixed in 2eca7be58e2990dabf716372c7734bee1e7d992f